cmd/commands/runtime_install.go

```diff
@@ -84,7 +83,10 @@ type (
 		IngressHost                    string
 		IngressClass                   string
 		InternalIngressHost            string
-		IngressController              ingressutil.IngressController
+		IngressController              routingutil.Controller
+		UseGatewayAPI                  bool
```

Comment:
If this is not a property that you get from outside and assign it yourself you should have it as private property.